src/graphql/Schema.js

```diff
@@ -3,21 +3,31 @@ const { merge } = require('lodash');
 const commonTypeDefs = require('./common/TypeDefs');
 const templateTypeDefs = require('./template/TypeDefs');
 const templateResolvers = require('./template/Resolvers');
+const deviceTypeDefs = require('./device/TypeDefs');
+const deviceResolvers = require('./device/Resolvers');
 
 
 const query = [`
-type Query {
-   #Get a template by Id
-    template(id: Int!): Template
-    #Checks if templates has Image Firmware and return a array with objects key-value, where key is a id template and value is a boolean.
-    #The value is true if the template has image firmware.
-    templatesHasImageFirmware(templatesId: [Int]!): [MapStringToString]
+  type Query {
+    #Get a template by Id
+      template(id: Int!): Template
+      #Checks if templates has Image Firmware and return a array with objects key-value, where key is a id template and value is a boolean.
+      #The value is true if the template has image firmware.
+      templatesHasImageFirmware(templatesId: [Int]!): [MapStringToString]
+      getDevices(page: PageInput, filter: FilterDeviceInput): [DeviceListPage]
+      getDeviceById(deviceId: String!): Device
+      getDeviceHistory(input: HistoryInput): [History]
+      getConfig(tenant: String!, user:String!): Config
+    }
+  type Mutation {
```

Comment:
I think It's better remove everything related with  user and config in this PR. :)